src/components/data/LocalVariablesContent.js

```diff
@@ -151,11 +189,12 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
               onChange: setEditKey
             }) :
             renderDataCell(amendedAttributes[rowIndex][0], googleProject)
-        }, {
+        },
+        {
```

Comment:
Could you revert this line?
```suggestion
        }, {
```

Reply:
Done.